pkg/tests/dbaas_operator_tests.go

```diff
@@ -37,41 +37,39 @@ var _ = Describe("Rhoda e2e Test", func() {
 		It("Should pass when operator installation is validated", func() {
 			fmt.Println("checking operator installation")
 			// Make sure the CRD exists
-			_, err = apiextensions.ApiextensionsV1().CustomResourceDefinitions().Get(context.TODO(), "dbaasplatforms.dbaas.redhat.com", meta.GetOptions{})
-			if err != nil {
-				AbortSuite(err.Error())
-			}
+			_, err := apiextensions.ApiextensionsV1().CustomResourceDefinitions().Get(context.TODO(), "dbaasplatforms.dbaas.redhat.com", meta.GetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
 
 	Context("Get all the providers and loop through it to create Secrets and Inventory", func() {
 		var providers []ProviderAccount
-		//Get ci-secret's data
+		var ciSecret *core.Secret
+		var client k8sClient.Client
 		clientset, err := kubernetes.NewForConfig(config)
 		Expect(err).NotTo(HaveOccurred())
-		ciSecret, err := clientset.CoreV1().Secrets("osde2e-ci-secrets").Get(context.TODO(), "ci-secrets", meta.GetOptions{})
-		if err != nil {
-			AbortSuite(err.Error())
-		}
-
-		//get the list of providers by getting providerList secret
-		if providerListSecret, ok := ciSecret.Data["providerList"]; ok {
-			//fmt.Printf("providerListSecret = %s, ok = %v\n", providerListSecret, ok)
-			providerNames := strings.Split(string(providerListSecret), ",")
-			providers = getProvidersData(providerNames, ciSecret.Data)
-		} else {
-			AbortSuite("ProviderList secret was not found")
-		}
 
-		//add dbaas scheme for inventory creation
-		scheme := runtime.NewScheme()
-		err = dbaasv1alpha1.AddToScheme(scheme)
-		Expect(err).NotTo(HaveOccurred())
+		//Set config and get ci-secret's data
+		It("Getting ci-secret and providerList secret", func() {
+			ciSecret, err = clientset.CoreV1().Secrets("osde2e-ci-secrets").Get(context.TODO(), "ci-secrets", meta.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			//get the list of providers by getting providerList secret
+			if providerListSecret, ok := ciSecret.Data["providerList"]; ok {
+				//fmt.Printf("providerListSecret = %s, ok = %v\n", providerListSecret, ok)
+				providerNames := strings.Split(string(providerListSecret), ",")
+				providers = getProvidersData(providerNames, ciSecret.Data)
+			} else {
+				Expect(ok).To(BeTrue(), "ProviderList secret was not found")
```

Comment:
Please use the exact secret name that you are looking for... in this case, `providerList`?

Reply:
Done